webpack.config.js

```diff
@@ -69,17 +93,13 @@ module.exports = () => {
     plugins: [
       new CleanWebpackPlugin(),
       new HtmlWebpackPlugin({
-        template: './public/index.html',
-        favicon: './src/client/app/assets/favicon/favicon.ico',
+        template: path.join('./public', templatePath, 'index.html'),
+        favicon: `./src/client/app/${assetResolveDir}/favicon/favicon.ico`,
         chunksSortMode: 'none',
-        meta: {
-          // Open Graph protocol meta tags
-          'og:title': 'Go.gov.sg',
-          'og:type': 'article',
-          'og:description': 'The official Singapore government link shortener',
-          'og:image':
-            'https://s3-ap-southeast-1.amazonaws.com/gosg-public/gosg-landing-meta.jpg',
-        },
+        meta: metaVariant,
+      }),
+      new webpack.DefinePlugin({
+        ASSET_VARIANT: JSON.stringify(process.env.ASSET_VARIANT),
```

Comment:
```suggestion
        'process.env.ASSET_VARIANT': JSON.stringify(process.env.ASSET_VARIANT),
```
And then do the suggested change mentioned earlier